internal/blockchain/fabric/fabric.go

```diff
@@ -883,8 +831,27 @@ func (f *Fabric) getNetworkVersion(ctx context.Context, chaincode string) (int,
 	return int(output.Result.(float64)), nil
 }
 
-func (f *Fabric) NetworkVersion() int {
-	f.fireflyContract.mux.Lock()
-	defer f.fireflyContract.mux.Unlock()
-	return f.fireflyContract.networkVersion
+func (f *Fabric) GetAndConvertDeprecatedContractConfig(ctx context.Context) (location *fftypes.JSONAny, fromBlock string, err error) {
+	// Old config (attributes under "fabconnect")
+	chaincode := f.fabconnectConf.GetString(FabconnectConfigChaincodeDeprecated)
+	if chaincode != "" {
+		log.L(ctx).Warnf("The %s.%s config key has been deprecated. Please use %s.%s instead",
+			FabconnectConfigKey, FabconnectConfigChaincodeDeprecated,
+			FireFlyContractConfigKey, FireFlyContractChaincode)
+	} else {
+		return nil, "", i18n.NewError(ctx, coremsgs.MsgMissingPluginConfig, "chaincode", "blockchain.fabric.fabconnect")
+	}
+	fromBlock = string(core.SubOptsFirstEventOldest)
+
+	fabricOnChainLocation := &Location{
+		Chaincode: chaincode,
+		Channel:   f.defaultChannel,
+	}
+
+	normalized, err := json.Marshal(fabricOnChainLocation)
+	if err == nil {
```

Comment:
If err is non-nil it should be surfaced

Reply:
we decided to just swallow the error, instead of surfacing it. Since we don't actually think one will be thrown.